pkg/prettyerr/text_formatter_test.go

```diff
@@ -17,52 +17,51 @@ func TestTextFormatter(t *testing.T) {
 		Code:    "GenericError",
 	}
 
-	want :=
-		"File config is missing.\n" +
-			"\tfilename: /etc/foo/config.yml\n" +
-			"Something happened, I'm sorry.\n"
-
 	stack := Errors{missingConfig, genericError}
-	format := "{{ .Message }}\n{{ range $key, $value := .Details }}\t{{ $key }}: {{ $value }}\n{{ end }}"
-
-	got, err := TextFormatter(stack, format)
-	if err != nil {
-		t.Errorf("TextFormatter() returned error: %v", err)
-	}
-
-	if got != want {
-		t.Errorf("TextFormatter(). Got:\n%v\nWant:\n%v", got, want)
-	}
-
-	invalidFormat := "{{ .Messag }}\n{{ range $key, $value := .Details }}\t{{ $key }}: {{ $value }}\n{{ end }}"
-	want = "failed to execute template for 0: template: node:1:3: executing \"node\" at <.Messag>: can't evaluate field Messag in type prettyerr.Error\n" +
-		"File config is missing.\n" +
-		"\tfilename: \"/etc/foo/config.yml\"\n" +
-		"failed to execute template for 1: template: node:1:3: executing \"node\" at <.Messag>: can't evaluate field Messag in type prettyerr.Error\n" +
-		"Something happened, I'm sorry.\n"
 
-	got, err = TextFormatter(stack, invalidFormat)
-	if err != nil {
-		t.Errorf("TextFormatter() returned error: %v", err)
-	}
-
-	if got != want {
-		t.Errorf("TextFormatter(). \nGot:\n%v\nWant:\n%v", got, want)
+	tests := []struct {
+		name   string
+		format string
+		want   string
+	}{
+		{
+			"No TextFormatter error",
+			"{{ .Message }}\n{{ range $key, $value := .Details }}\t{{ $key }}: {{ $value }}\n{{ end }}",
+			"File config is missing.\n" +
+				"\tfilename: /etc/foo/config.yml\n" +
+				"Something happened, I'm sorry.\n",
+		},
+		{
+			"Fields error",
+			"{{ .Messag }}\n{{ range $key, $value := .Details }}\t{{ $key }}: {{ $value }}\n{{ end }}",
+			"failed to execute template for 0: template: node:1:3: executing \"node\" at <.Messag>: can't evaluate field Messag in type prettyerr.Error\n" +
+				"File config is missing.\n" +
+				"\tfilename: \"/etc/foo/config.yml\"\n" +
+				"failed to execute template for 1: template: node:1:3: executing \"node\" at <.Messag>: can't evaluate field Messag in type prettyerr.Error\n" +
+				"Something happened, I'm sorry.\n",
```

Comment:
It is possible to define a multiline string using backticks:

```go
value := `failed to execute template for 1:
template: node:1:3: executing \"node\" at`
```

Reply:
Back ticks fail to interpret \n and \t. If newline and tabs are used directly, readability of test file is hindered.

![image](https://user-images.githubusercontent.com/32382590/95123444-99523d00-076f-11eb-9efe-6d9621d5e932.png)

Reply:
@jeroenrinzema shall I push without backticks?

Reply:
agreed, let's keep it as it is.

Reply:
@jeroenrinzema I have amended the commit